packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.types.ts

```diff
@@ -41,6 +41,13 @@ export interface IColorPickerProps {
    */
   alphaSliderHidden?: boolean;
 
+  /**
+   * When useTransparencySlider is true, this will change the alpha slider to transparency slider.
+   * Alpha value represents the opacity of the color whereas transparency reperesnts the tranparentness of the color,
+   * i.e. 30% transparent color has 70% opaqueness.
+   */
+  useTransparencySlider?: boolean;
```

Comment:
I still feel showAlphaAsTransparency

Reply:
accepted the @ecraig12345  suggestion to make it less verbose, i believe useTransparencySlider is less verbose and precise.